src/style.css

```diff
@@ -1 +1,104 @@
 /* Styles go here */
+body {
+  background: #e5e5e5;
+  margin: 0;
+}
+
+@media (max-width: 1024px) {
```

Comment:
No needs to use media queries for this task. You can just apply max-width to your header and place it at the center